api/spec/support/next_instance_of.rb

```diff
@@ -0,0 +1,38 @@
+# Helper services with prepended ServiceModule
```

Comment:
@btolarz please move this into https://github.com/spree/spree/tree/master/core/lib/spree/testing_support

Reply:
Done